include/download_client.rst

```diff
@@ -118,8 +118,12 @@ The following snippet illustrates how to provision a TLS certificate, associate
 		 * calling download_client_connect().
 		 */
 
+		download_client_cfg config = {
+			.sec_tag = sec_tag,
+		};
+
 		/* Pass the security tag to the download library */
```

Comment:
This comment is not very accurate at this point, I'd just drop it.

Reply:
I moved it up aboe `download_client_cfg config = {...` and rephrased it slightly.